backend/src/constants.ts

```diff
@@ -61,13 +68,13 @@ export const PATHS = {
 
   MAC_USER_BINARIES: path.join(os.homedir(), '.remoteit/bin'),
   MAC_USER_SETTINGS: path.join(os.homedir(), '.remoteit'),
-  MAC_ADMIN_BINARIES: '/usr/local/bin',
+  MAC_ADMIN_BINARIES: './resources/mac',
```

Comment:
Could this path be the same on both mac and win?
```suggestion
  MAC_ADMIN_BINARIES: './resources',
```